composer.json

```diff
@@ -31,26 +31,25 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-servicemanager": "^3.14.0",
```

Comment:
In this package, I notice that we didn't have any `contianer-interop` to `psr/container`: we **could** in theory just add a conflict clause with earlier versions, instead of introducing a forced dependency :thinking:

Reply:
Good point - the only imports were in test cases. I'll do that now.